BrainPortal/app/views/groups/new.html.erb

```diff
@@ -59,7 +59,12 @@
     to this project (but editors are always allowed to do so):
     <%= f.check_box :not_assignable %>
 
-    <% if current_user.has_role?(:normal_user) %>
+    <p>
+    Make public, so all user can access it. Be <b> careful </b> with this option! You can always make project to public latter on:
```

Comment:
Fix text, you're missing articles: `Make the project public, so that all users can access the files. Be careful with this option! You can always make the project public later on.`

Reply:
`all _the_ user` I guess ? are any other articles or grammar issues missed?